<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRecords</class>
 <widget class="QDialog" name="EditRecords">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,0,0,0">
   <item row="1" column="3">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="valuesTable"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="restoreDefaultsButton">
     <property name="text">
      <string>Restore Defaults</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
